From 1c079875d1a84042bab0ca63762aa1cd15531f77 Mon Sep 17 00:00:00 2001
From: Fs <Fsu0413@vip.qq.com>
Date: Sun, 28 Jul 2024 10:29:49 +0800
Subject: [PATCH 2/3] use standard include guard

---
 p256.h | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/p256.h b/p256.h
index d979ba8..cb3b511 100644
--- a/p256.h
+++ b/p256.h
@@ -1,4 +1,5 @@
-#pragma once
+#ifndef OREPARAZ_P256_H
+#define OREPARAZ_P256_H
 
 typedef enum {
   P256_SUCCESS = 1,
@@ -18,3 +19,5 @@ p256_ret_t p256_verify(uint8_t *msg, size_t msg_len, uint8_t *sig, const uint8_t
 #ifdef __cplusplus
 }
 #endif
+
+#endif
-- 
2.43.0.windows.1

